.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  push:
+    tags:        
+      - '*'
+
+jobs:
+  docker:
+    name: Release Docker
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: iotaledger/bee
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.IOTALEDGER_DOCKER_USERNAME }}
+          password: ${{ secrets.IOTALEDGER_DOCKER_PASSWORD }}
+
+      - name: Build and push to Dockerhub
+        uses: docker/build-push-action@v2
+        with:
+          file: ./docker/Dockerfile
```

Comment:
Is this correct! I think the Dockerfile is in bee-node!

Reply:
Puh, nice catch. I'll fix that real quick.